File: Proyectos python (4)/numero repetido.py
cont=0
nr=0
num=0
des=0
print("Programa que va a determinar cuantas veces aparece un numero ingresado por el usuario")
nr=(int(input("Ingrese el numero que se va a repetir:\n")))
while des != 2:
    num=(int(input("Ingrese un numero: \n")))
    if(num == nr):
        cont=cont+1
    else:
        num=(int(input("Ingrese un numero: \n")))
    des=(int(input("¿Desea ingresar mas numeros? Si=1, No=2\n")))

print("El numero que mas se repite es: ",nr, "\nCon una cantidad de: ",cont+1," veces.")
    

File: Proyectos C++ (4)/numero repetido.cpp
#include <iostream>
#include<locale.h>

using namespace std;

int main(int argc, char *argv[]) {
setlocale(LC_CTYPE, "spanish"); 
int num,cont,ds,cn;
 
 
 cout<<"Programa que va a determina cuantas veces se repite un numero ingresado por el usuario "<<endl;
 cout<<"\nIngrese el numero que se va a repetir : ";
 cin>>cn;
 
 cont=0;
  
 while(ds!=2){
	cout<<"Ingrese un número: "<<endl;
	cin>>num;
	if(num == cn)
	{
		cont++;
    }
    else
    {
    cout<<"Ingrese un número: "<<endl;
	cin>>num;	
	}
	cout<<"¿Desea ingresar mas números? Si=1, No=2: "<<endl;
 	cin>>ds;
 
 
 
 
} 
 cout<<"El número que mas se repite es: "<<cn<<endl;
 cout<<"Con una cantidad de: "<<cont++<<" veces"<<endl;

 
 return 0;
}

